Assets/Storage/DBFile.php

```diff
@@ -528,4 +528,19 @@ public function canViewFile() {
 				->getStore()
 				->canView($this->Filename, $this->Hash);
 	}
+
+	public function PreviewLink($action = null) {
```

Comment:
Currently used as follows:

```php
$width = (int)Image::config()->get('asset_preview_width');
$previewLink = Convert::raw2att($this->ScaleMaxWidth($width)->PreviewLink());
$image = "<img src=\"{$previewLink}\" class=\"editor__thumbnail\" />";
```

We already have the following in `ImageManipulation.php`:

```php
public function PreviewThumbnail() {
	$width = (int)Config::inst()->get(get_class($this), 'asset_preview_width');
	return $this->ScaleWidth($width)  ?: $this->IconTag();
}
```

Took me a while to understand why this method was necessary: Because we want to add a class to the image tag, hence can't use `IconTag()`. And this method was added to `DBFile` rather than `ImageManipulation` because its technically not a manipulation.

All good, apart from some clarification in the commit message which could've simplified the peer review.

Oh, and PHPDoc for the method please.